README.rst

```diff
@@ -180,7 +180,7 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
-    @software{pygmt_2023_7772533,
+    @software{pygmt_2023_8303186,
       author       = {Uieda, Leonardo and
```

Comment:
Do we keep the same order list for the authors? Output of commits made on main branch at 1d8c0fdfd1c986147641aaf0cdb4e9e47af988ae to first commit at e3c542a173ad6f74d5072326d07275662249f288 (`git shortlog 1d8c0f...e3c542a -sne`) shows this after I merge the same author names:

```
544  Dongdong Tian
255  Leonardo Uieda
216  Wei Ji
191  Will Schlitzer
169  dependabot[bot]
130  Yvonne Fröhlich
124  Michael Grund
109  Max Jones
 33  Yao Jiayuan
 15  Liam Toney
  7  yohaimagen
  4  Tong
  3  kmaterna
  2  Abhishek Anant
  2  Amanda Leaman
  2  Andre L. Belem
  2  Andrey Shmakov
  2  Claudio Satriano
  2  Conor Bacon
  2  Jamie J Quinn
  2  MichaeINeumann
  2  Noor Buchi
  2  Shivani chauhan
  2  Tyler Newton
  2  alperen-kilic
  2  carocamargo
  2  daroari
  2  obaney
  2  srijac
  1  Alicia Ngoc Diep Ha
  1  Andrés Ignacio Torres
  1  Becky Salvage
  1  Brook
  1  Cody Woodson
  1  Emily McMullan
  1  Jack Beagley
  1  Julius Busecke 
  1  Kadatatlu Kishore
  1  Loïc Houpert
  1  Malte Ziebarth
  1  Mark Wieczorek
  1  Philipp Loose
  1  Sean H
  1  Soham Banerjee
  1  Tawanda
  1  Vitor Gratiere Torres
  1  mdtanker
  1  shahid-0
  1  teriyakiSauce23
```

Reply:
Based on the counts of commits, I think we should at least make two changes:

1. Move "Yvonne Fröhlich" before "Michael Grund"
2. Move "Yao Jiayuan" before "Liam Toney"

As for the order between Leo and me, I'm not sure about this, considering that Leo created this package.

Reply:
> Move "Yvonne Fröhlich" before "Michael Grund"

Ok with this. I've been meaning to ask @yvonnefroehlich if she would like to be a maintainer actually, she's been helping a lot with the documentation and doing a lot of good quality reviews lately.

> Move "Yao Jiayuan" before "Liam Toney"

I don't think either of them have made a commit since 2021 (< v0.4.1), so not sure why we should change the order now... Liam did do quite a bit of work on the ROSES 2021 and 2020 videos, and I it was somewhat agreed at https://github.com/GenericMappingTools/pygmt/pull/1414#discussion_r683333747 to keep the order this way?

> As for the order between Leo and me, I'm not sure about this, considering that Leo created this package.

Leo said at https://github.com/GenericMappingTools/pygmt/pull/1594#discussion_r740248554 that he was happy for you to go first, and I'd say you deserve it more now :slightly_smiling_face: But we can also leave it if you feel shy, and put your name first on the paper :wink:

Reply:
> > Move "Yvonne Fröhlich" before "Michael Grund"
> 
> Ok with this. I've been meaning to ask @yvonnefroehlich if she would like to be a maintainer actually, she's been helping a lot with the documentation and doing a lot of good quality reviews lately.

Yes, she's well qualified. 

> > Move "Yao Jiayuan" before "Liam Toney"
> 
> I don't think either of them have made a commit since 2021 (< v0.4.1), so not sure why we should change the order now... Liam did do quite a bit of work on the ROSES 2021 and 2020 videos, and I it was somewhat agreed at [#1414 (comment)](https://github.com/GenericMappingTools/pygmt/pull/1414#discussion_r683333747) to keep the order this way?

Sounds reasonable. Let's keep it unchanged.
 
> > As for the order between Leo and me, I'm not sure about this, considering that Leo created this package.
> 
> Leo said at [#1594 (comment)](https://github.com/GenericMappingTools/pygmt/pull/1594#discussion_r740248554) that he was happy for you to go first, and I'd say you deserve it more now 🙂 But we can also leave it if you feel shy, and put your name first on the paper 😉

Aha, I didn't realize that. It's my honor to be listed as the first one. BTW, you're really good at digging history 😉

Reply:
Ok, done at 852b025b26e14d6b5e327858a4312591560ef914

Reply:
>>> Move "Yvonne Fröhlich" before "Michael Grund"

>> Ok with this. I've been meaning to ask @yvonnefroehlich if she would like to be a maintainer actually, she's been helping a lot with the documentation and doing a lot of good quality reviews lately.

> Yes, she's well qualified.

First of all thanks a lot for this positive feedback, @weiji14 and @seisman :blush:. I'm glad to hear this! I am really thankful for contributing to this project; I learned a lot. So, I would be pleased to become a maintainer of PyGMT. Hopefully the other @GenericMappingTools/pygmt-maintainers are also fine with this.
I feel I should have the time to help with the documentation and reviewing PRs in the future, but for more complicated code-related tasks I would need more (free) time, which is currently not very likely because of the workload of my PhD projects.

Just one comment regarding the commit list above: Looking at the commit numbers of the single contributors at https://github.com/GenericMappingTools/pygmt/graphs/contributors there is partly a significant difference to `git shortlog 1d8c0f...e3c542a -sne` (I can reproduce @weiji14 numbers). Looking at the total number of commits made so far to this repro, it seems that GitHub additionally includes / counts co-authorship?

Reply:
> > > > Move "Yvonne Fröhlich" before "Michael Grund"
> 
> > > Ok with this. I've been meaning to ask @yvonnefroehlich if she would like to be a maintainer actually, she's been helping a lot with the documentation and doing a lot of good quality reviews lately.
> 
> > Yes, she's well qualified.
> 
> First of all thanks a lot for this positive feedback, @weiji14 and @seisman 😊. I'm glad to hear this! I am really thankful for contributing to this project; I learned a lot. So, I would be pleased to become a maintainer of PyGMT. Hopefully the other @GenericMappingTools/pygmt-maintainers are also fine with this. I feel I should have the time to help with the documentation and reviewing PRs in the future, but for more complicated code-related tasks I would need more (free) time, which is currently not very likely because of the workload of my PhD projects.

No worries, feel free to help out as much (or as little) as you can, PhD comes first of course! If you can open up a PR to modify [team.md](https://github.com/GenericMappingTools/pygmt/blob/main/doc/team.md), we can continue on from there.

> Just one comment regarding the commit list above: Looking at the commit numbers of the single contributors at https://github.com/GenericMappingTools/pygmt/graphs/contributors there is partly a significant difference to `git shortlog 1d8c0f...e3c542a -sne` (I can reproduce @weiji14 numbers). Looking at the total number of commits made so far to this repro, it seems that GitHub additionally includes / counts co-authorship?

Yes, it might be that GitHub includes co-authored commits, whereas `git log -sne` only counts the main author.

Reply:
> No worries, feel free to help out as much (or as little) as you can, PhD comes first of course!

Sounds good 🙂!

> If you can open up a PR to modify [team.md](https://github.com/GenericMappingTools/pygmt/blob/main/doc/team.md), we can continue on from there.

Please see PR #2655.